Resources/Prototypes/DeltaV/Entities/Structures/Doors/Windoors/windoor.yml

```diff
@@ -21,6 +21,41 @@
   id: WindoorSecureParamedicLocked
   suffix: Paramedic, Locked
   components:
+  - type: AccessReader
```

Comment:
Remove this. Only need `ContainerFill`

Reply:
This is the paramedic windoor, but I'll bunch it in with the justice PR if it won't get stricken because of it

Reply:
Go ahead and fix all the ones that need to be